db/migrate/20171018174809_number_of_jobs_to_integer.rb

```diff
@@ -0,0 +1,23 @@
+class NumberOfJobsToInteger < ActiveRecord::Migration[5.1]
+  def up
+    add_column :medicaid_applications, :new_number_of_jobs, :integer
+
+    safety_assured do
+      execute <<~SQL
+        UPDATE medicaid_applications
+        SET new_number_of_jobs=1
+        WHERE number_of_jobs='1 job';
+        UPDATE medicaid_applications
+        SET new_number_of_jobs=2
+        WHERE number_of_jobs='2 jobs';
+        UPDATE medicaid_applications
+        SET new_number_of_jobs=3
+        WHERE number_of_jobs='3 or more jobs';
+      SQL
```

Comment:
Lost the semicolons and thought this was one long query. 

Thoughts on splitting them up into several `executes "..."`? Or some newlines in between?

Reply:
sure! done c38c51e